src/components/FeatureStore/FeatureStorePageTabs/FeatureStorePageTabs.jsx

```diff
@@ -32,6 +32,7 @@ import {
 } from '../../../constants'
 
 const FeatureStorePageTabs = () => {
+  // To delete after new navbar is merged
```

Comment:
Not sure why we need this comment. Looks like we still have tabs for Feature Store